robotbase/src/main/java/frc/robot/commands/elevator/ElevatorSetRotations.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.commands.elevator;
+
+import edu.wpi.first.units.measure.Angle;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Robot;
+
+public class ElevatorSetRotations extends Command {
```

Comment:
This command helps highlight the utility of the Units library in robotlib, but is not really necessary in practical robot code, let's remove this command. For all practical purposes, we should be setting the elevator using distance over rotations.